crates/biome_plugin_loader/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
```

Comment:
It's possible you created the crate manually or with cargo. If so you, you need to update manually `knope.toml` and add the crate. We have `just new-crate` that updates the file automatically

Reply:
Done